README.md

```diff
@@ -389,6 +389,215 @@ In the previous examples, Docker secrets files were used to specify admin creden
 
 Using Docker secrets, username and password information are secured when transmitting the sensitive data from Docker host to Docker containers. To prevent any attacks, the login information is not available as an environment variable. However, these values are stored in a text file and persisted in an in-memory file system inside the container. We recommend that you delete the Docker secrets information once the cluster is up and running.
 
+### How to use Docker Secrets with Docker Stack
+
+1. Run below to initialize swarm setup:
+
+```
+  $docker swarm init
+```
+2. Create secrets using the below commands:
+ - Create mldb_admin_username_v1.txt file to add mldb_admin_username_v1 secret for admin username using the following command:
```

Comment:
Create `mldb_admin_username_v1.txt` file to add the `mldb_admin_username_v1` secret for the admin username using the following command: